music_assistant/providers/chromecast/helpers.py

```diff
@@ -82,7 +82,11 @@ def fill_out_missing_chromecast_info(self, zconf: Zeroconf) -> None:
         self.is_dynamic_group = self.uuid in dynamic_groups
         if self.uuid in multichannel_groups:
             self.is_multichannel_group = True
-        elif multichannel_groups:
+        elif (
+            multichannel_groups
+            and self.cast_type != "group" 
```

Comment:
```suggestion
            and self.cast_type != "group"
```

Remove the white space